### ISSUE TITLE
Passkey support

### ISSUE THREAD

**Issue by zenlord** [linux-pam/linux-pam#616]
Hi, not an issue per se - just a polite question to gauge whether support for [passkeys](https://www.passkeys.io/) is being considered in linux-pam?

**Comment by ldv-alt** [comment]
Isn't https://github.com/Yubico/pam-u2f the pam module you're looking for?

**Comment by robin-thoni** [comment]
I'm not sure this module implements passkeys, but just U2F. Did I get it wrong?

Also, even though I don't think it's possible with the current pam interface, having passkeys QR Code support would be neat

**Comment by zenlord** [comment]
@robin-thoni I believe you are right. "Passkeys support" is more than U2F. Yubico keys do implement a specific type of passkeys, but using passkeys can be implemented without the need for additional hardware (assuming that a smartphone is part of the default hardware).

QR codes is one way, but a proof of concept that I came across some weeks ago lets the browser communicate with the smartphone using bluetooth.

**Comment by robin-thoni** [comment]
Right, BLE, USB and NFC passkeys support would be nice, especially since they do not require any change in the UI and/or pam interface. The protocols do not seem quite obvious to implement, though :/

**Comment by thkukuk** [comment]
With the complexity of passkeys I don't see that this will be implemented in Linux-PAM. It's more likely, and makes much more sense, to start an own project for this. Like for many other PAM modules, there is no reason or benefit to add them to Linux-PAM.
